src/main/java/dk/cngroup/lentils/dto/ClientSkipCypherActionDTO.java

```diff
@@ -0,0 +1,19 @@
+package dk.cngroup.lentils.dto;
+
+public class ClientSkipCypherActionDTO {
```

Comment:
Je pro tohle potřeba DTO? Nestačí poslat ID v URL adrese?

Nebo se to tlačítko pro skip chová na pozadí jako formulář?

Reply:
stačí jen ID v URL adrese, upravim